requirements.txt

```diff
@@ -32,6 +32,7 @@ google-cloud-bigquery~=2.34.4
 google-cloud-iam==2.9.0
 google-cloud-run==0.5.0
 google-cloud-storage==2.5.0
+kubernetes==25.3.0
```

Comment:
@dy46 Could you also add this package to `setup.py` `all` extra_requires?
I got this error when calling `/api/status` with latest docker image.
<img width="1667" alt="image" src="https://user-images.githubusercontent.com/80284865/210039920-142d29f4-c23f-4b7c-8ea4-a3bdcc961744.png">